synapse/async.py

```diff
@@ -228,6 +220,8 @@ def _runJob(self, job):
         '''
         task = job[1].get('task')
         if task == None:
+            # TODO This attribute is not set, a bad tufo
+            # sent to _runJob will have unexpected behavior.
```

Comment:
I just wanted to note that the attribute is currently not set by anything; so this code path would throw an AttributeError.